### ISSUE TITLE
[$40][MSFT-107] Challenges page: Datepicker: The instructions to obtain callendar shortcuts are ambiguous for screen reader users.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2811]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 2. Ensure the screen readr is switched on. 3. Activate the filter. 4. Press the tab key till you arrive at the start date field. 5. Press the '?' key from the keyboard as mentioned in the instructions.6. Notice that no changes are announced by the screen readers.

**Expected Result**
There should be button 'callendar keyboard shortcuts' which when activated should show the keyboard shortcuts. Hence the instructions given are not clear.

**Actual Result**
Screen reader doesn't announce any keyboard shortcuts to work with the callendar.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 3. Understandable
**WCAG Level**: A
**WCAG Success Criterion**: 3.3.2 Labels or Instructions
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2013

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096051 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096051 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096051 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097824 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097824 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097824 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL : https://test-community-app.topcoder-dev.com/challenges

**Comment by drasticdpk** [comment]
Verified on
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097824<br/><br/>```This is an automated message for crazyk via Topcoder X```
